src/components/app/custom-els/FileDrop/styles.css

```diff
@@ -0,0 +1,7 @@
+file-drop {
+  display: block;
+  overflow: hidden;
+  touch-action: none;
+  height:100%;
+  width:100%;
+}
```

Comment:
These don't feel like styles everyone would want, so they shouldn't be here. Perhaps with the exception of `display: block`.